parent/pom.xml

```diff
@@ -100,6 +100,7 @@
 
     <!-- Dependency Versions -->
     <jackson.version>2.9.9</jackson.version>
```

Comment:
I think we also need to update this version to at least 2.9.9.2. 

See https://dev.azure.com/azure-sdk/internal/_componentGovernance/106501/alert/1778976?typeId=390820

Reply:
Never mind I see that CG issue is only for databind.